From 3c78d9ee52b742073257caec31d6d0ad843cd66c Mon Sep 17 00:00:00 2001
From: Haohui Mai <wheat9@apache.org>
Date: Fri, 24 Oct 2014 10:50:16 -0700
Subject: [PATCH 211/596] HDFS-7283. Bump DataNode OOM log from WARN to ERROR.
 Contributed by Stephen Chu.

(cherry picked from commit b3d8a642a938da9de680b479585a7c2014b8965c)
(cherry picked from commit ad0972168701f5c4d69cdccbf68750f79fcaf745)
---
 .../hdfs/server/datanode/DataXceiverServer.java    |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DataXceiverServer.java b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DataXceiverServer.java
index a1f766d..9bf95eb 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DataXceiverServer.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DataXceiverServer.java
@@ -161,7 +161,7 @@ public void run() {
         // DataNode can run out of memory if there is too many transfers.
         // Log the event, Sleep for 30 seconds, other transfers may complete by
         // then.
-        LOG.warn("DataNode is out of memory. Will retry in 30 seconds.", ie);
+        LOG.error("DataNode is out of memory. Will retry in 30 seconds.", ie);
         try {
           Thread.sleep(30 * 1000);
         } catch (InterruptedException e) {
-- 
1.7.9.5

